### ISSUE TITLE
Ignore tests in the __test_runner sub folder

### ISSUE THREAD

**Issue by Fox32** [google/test_runner.dart#16]
I noticed that after a crash or canceling of the test runner the ```__test_runner``` sub folder is not cleaned up. I the next run it also finds the tests in the sub folder and therefore runs every test twice:

```
Running all tests...
Test suite passed: grammar_test.dart
...
Test suite passed: __test_runner/grammar_test.dart
````

**Comment by nicolasgarnier** [comment]
Mhh strange :D I thought I was making sure they would not be ran twice. Let me look into this.

**Comment by nicolasgarnier** [comment]
It seems that's only happening under windows because in dart_project.dart we missed this:

            TestRunnerCodeGenerator.GENERATED_TEST_FILES_DIR_NAME + "/")) {

**Comment by nicolasgarnier** [comment]
I just published a new version that should fix this (0.2.11+4). Please re-open if this did not fix it.

**Comment by Fox32** [comment]
Thanks, works fine now!
